static/app/views/settings/account/accountClose.tsx

```diff
@@ -147,16 +150,17 @@ function AccountClose() {
       </Alert>
 
       <Panel>
-        <PanelHeader>{t('Remove the following organizations')}</PanelHeader>
+        <PanelHeader>{t('Delete the following organizations')}</PanelHeader>
         <PanelBody>
           <PanelAlert type="info">
+            <strong>{t('ORGANIZATIONS WITH CHECKED BOXES WILL BE DELETED')}</strong>
+            <br />
             {t(
               'Ownership will remain with other organization owners if an organization is not deleted.'
             )}
             <br />
-            {tct(
-              "Boxes which can't be unchecked mean that you are the only organization owner and the organization [strong:will be deleted].",
-              {strong: <strong />}
+            {t(
+              "Boxes which can't be unchecked mean that you are the only organization admin and will be deleted."
```

Comment:
only owners have the power to delete organizations, let's keep that copy here

```suggestion
              "Boxes which can't be unchecked mean that you are the only organization owner and will be deleted."
```